src/api/wallet.ts

```diff
@@ -66,15 +67,16 @@ export const signAndSubmitTransaction = async (
       transactionPayload,
     );
     if ("hash" in response) {
+      await client.waitForTransaction(response["hash"]);
```

Comment:
I wonder if that something that needs to be happen on the wallet side, and we only should get the success/failure response?
@kent-white any thoughts?

Reply:
Agree. This is a walk-around for now. I think eventually we want to have this from the wallet. Maybe something like `window.aptops.waitForTransaction()`.